### PR TITLE
Feature: Dashboard Menu Manager

### PR DESCRIPTION
**Pull request by aihimel**
### All Submissions:

* [x] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [x] My code satisfies feature requirements
* [x] My code is tested
* [x] My code passes the PHPCS tests
* [x] My code has proper inline documentation
* [x] I've included related pull request(s) (optional)
* [ ] I've included developer documentation (optional)
* [x] I've added proper labels to this pull request

### Related Pull Request(s)

* https://github.com/getdokan/dokan-pro/pull/2791

### Closes 

* Closes https://github.com/getdokan/dokan-pro/issues/2482

### How to test the changes in this Pull Request:

* Check https://github.com/getdokan/dokan-pro/issues/2482

### Changelog entry

*Title*

Detailed Description of the pull request. What was previous behaviour
and what will be changed in this PR.

### Before Changes

Describe the issue before changes with screenshots(s).

### After Changes

Describe the issue after changes with screenshot(s).

### Feature Video (optional)

Link of detailed video if this PR is for a feature.

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

### CODE REVIEW

**Review by nurul-umbhiya — APPROVED** [review]

**Comment by Mohaiminulislam1989** [comment]
@StalinDurjo please check again if this pr fix the bug.

**Comment by StalinDurjo** [comment]
@Mohaiminulislam1989 bhai working on it, will let you know once done.

**Comment by Mohaiminulislam1989** [comment]
Thank you

**Comment by StalinDurjo** [comment]
@Mohaiminulislam1989 Bhai, for some reason the code is not working on our devices. Aftab bhai has checked the issue and is looking into it.

**Comment by Mohaiminulislam1989** [comment]
Any update?

**Review by nurul-umbhiya — CHANGES_REQUESTED** [review]

File: includes/Shortcodes/Dashboard.php [hunk 1]
```diff
@@ -32,7 +32,12 @@ public function render_shortcode( $atts ) {
 
         ob_start();
 
-        if ( isset( $wp->query_vars['products'] ) ) {
+        /**
+         * Filter query var before rendering dokan vendor shortcode
+         */
+        $query_vars = apply_filters( 'dokan_load_custom_template_before_action', $wp->query_vars );
```
[the remark this answers is not in the document]
Comment (aihimel) [reply includes/Shortcodes/Dashboard.php]: updated

File: src/admin/pages/Settings.vue [hunk 2]
```diff
@@ -72,7 +72,17 @@
...
                                     </div>
                                 </div>
-                                <p class="submit"><input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes" @click.prevent="saveSettings( settingValues[index], index )"></p>
+                                <p class="submit">
+                                    <input
+                                        type="submit"
+                                        name="submit"
+                                        id="submit"
+                                        class="button button-primary"
+                                        value="Save Changes"
```
[the remark this answers is not in the document]
Comment (aihimel) [reply src/admin/pages/Settings.vue]: Updated

**Pull request merged**